src/sentry/static/sentry/app/components/pills.jsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
-import classNames from 'classnames';
+import styled from 'react-emotion';
 
 class Pills extends React.Component {
   render() {
-    const {className, children, ...otherProps} = this.props;
-    return (
-      <div className={classNames('pills', className)} {...otherProps}>
-        {children}
-      </div>
-    );
+    const {children, ...otherProps} = this.props;
+    return <StyledPills {...otherProps}>{children}</StyledPills>;
   }
 }
 
+const StyledPills = styled('div')`
```

Comment:
So, our `<Pills>` React component is unnecessary now, we can just make this `Pills` and export this.

Reply:
It sounds like you are suggesting removing `<Pills>` completely? This will impact quite a few files or maybe I am mistaken?

Reply:
I'm suggesting removing the `Pills` class in this file and just having this styled component